internal/cloud/e2e/migrate_state_multi_to_tfc_test.go

```diff
@@ -310,8 +310,8 @@ func Test_migrate_multi_to_tfc_cloud_tags_strategy(t *testing.T) {
 				},
 			},
 			validations: func(t *testing.T, orgName string) {
-				wsList, err := tfeClient.Workspaces.List(ctx, orgName, tfe.WorkspaceListOptions{
-					Tags: tfe.String("app"),
+				wsList, err := tfeClient.Workspaces.List(ctx, orgName, &tfe.WorkspaceListOptions{
+					Tags: "app",
```

Comment:
Curious - I noticed that another string value changed to `[]tfe.WSIncludeOpt{tfe.WSCurrentRun}}` - would we want to do something similar here?

Reply:
Thanks for checking! Do you mean if `Tags` fields is a typed string like `Include` fields? If so, no, the `Tags` field is just a string in go-tfe. It used to be a pointer to a string, but is now is just a string: https://github.com/hashicorp/go-tfe/blob/main/workspace.go#L238

Reply:
Got it - thank you for the link. That makes sense 😄